app/jobs/resolution_proofing_job.rb

```diff
@@ -32,6 +32,7 @@ def perform(result_id:, encrypted_arguments:, trace_id:, should_proof_state_id:,
         applicant_pii, user, threatmetrix_session_id
       )
       log_threatmetrix_info(threatmetrix_result, user)
+      add_threatmetrix_proofing_component(user_id, threatmetrix_result)
```

Comment:
We might have some conflicts around this--I refactored this out into a `proof_lexisnexis_ddp_with_threatmetrix_if_needed` method

Reply:
is there a pr?

Reply:
https://github.com/18F/identity-idp/pull/6800

Reply:
ah cool... .pulling the latest

Reply:
done!  thanks!!!